src/easylink/graph_components.py

```diff
@@ -27,15 +35,83 @@ class Edge:
     target_node: str
     output_slot: str
     input_slot: str
+    filepaths: Optional[tuple[str]] = None
+
+    @classmethod
+    def from_graph_edge(cls, source, sink, edge_attrs) -> "Edge":
```

Comment:
I suggest renaming this `Edge` class to more naturally differentiate it from nx's edges.

Reply:
EdgeParams?